interchain-exchange/tutorial/09-walkthrough.md

```diff
@@ -18,8 +18,8 @@ Start the networks, as input use the `mars.yml` and `venus.yml` for each blockch
 
 ```bash
 # start networks
-starport serve -c mars.yml -r
-starport serve -c venus.yml -r
+starport serve -c interchain/mars.yml -r
```

Comment:
You must be in the directory of the app to serve it, unless you provide the `--path interchange` flag.
I would personally keep the original command to keep it simple and precise to the user to cd to the app path